plugins/defaultplugins/ifplugin/vppcalls/ip_container_vppcalls.go

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2017 Cisco and/or its affiliates.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vppcalls
+
+import (
+	"fmt"
+	"net"
+
+	govppapi "git.fd.io/govpp.git/api"
+	"github.com/ligato/cn-infra/logging"
+	"github.com/ligato/cn-infra/logging/measure"
+	"github.com/ligato/vpp-agent/plugins/defaultplugins/ifplugin/bin_api/ip"
+	"time"
+)
+
+// AddContainerIP calls IPContainerProxyAddDel VPP API with IsAdd=1
+func AddContainerIP(ifIdx uint32, addr *net.IPNet, isIpv6 bool, log logging.Logger, vppChan *govppapi.Channel, timeLog measure.StopWatchEntry) error {
+	// IPContainerProxyAddDelReply time measurement
+	start := time.Now()
+	defer func() {
+		if timeLog != nil {
+			timeLog.LogTimeEntry(time.Since(start))
+		}
+	}()
+
+	req := prepareMessageForVpp(ifIdx, addr, isIpv6, 1)
```

Comment:
do not use magic numbers. Either define contants (add = 1, del = 0) or use boolean

Reply:
ok